.github/workflows/run-tests.yml

```diff
@@ -1,6 +1,10 @@
 name: Run tests
 
-on: [push]
+on:
+  push:
+    paths-ignore:
+      - '**.md'
```

Comment:
This, and the line below sets CI to avoid running if we're making changes to documentation, or in the line below the `.gitignore` file. We almost definitely to not want to trigger a deploy and CI run in these cases so it makes sense to have them.